protos/sigstore_verification.proto

```diff
@@ -94,6 +108,7 @@ message ArtifactVerificationOptions {
         // Threshold: 1
         // Online verification: false
         // Disable: false
+        // Verify SET Timestamps: true
```

Comment:
```suggestion
```

Reply:
Since this option no longer controls SET timestamp verification

Reply:
Good catch!